db/seeds.rb

```diff
@@ -14,3 +15,7 @@
 
 Document.find_or_create_by(display_name: 'My Employment', document_type: 'Employment', client: client, created_at: 1.day.ago, intake: intake)
 Document.find_or_create_by(display_name: 'Identity Document', document_type: 'ID', client: client, created_at: 2.months.ago, intake: intake)
+
+OutgoingTextMessage.create!(client: client, body: "Hey client, how is it going", user: user, sent_at: 3.days.ago, to_phone_number: "+14155551212")
+OutgoingTextMessage.create!(client: client, body: "Hey client, how is it going", user: user, sent_at: 2.days.ago, to_phone_number: "+14155551212")
+OutgoingTextMessage.create!(client: client, body: "Hey client, how is it going", user: user, sent_at: DateTime.now, to_phone_number: "+14155551212")
```

Comment:
Non-blocking: Would it be helpful to have different messages so that when we view it locally it doesn't look like an error?

Reply:
Yeah - honestly I'd love ideas - in the absence of input, I'll go with

> Hey client, nice to meet you
> Hope you're having a good day
> Thanks and have a good night!

I'll merge this with the strings changed in approx 1h (not a deadline to you, just telling you)